app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -20,6 +20,18 @@ def self.event_monitor_class
     ManageIQ::Providers::Openshift::ContainerManager::EventCatcher
   end
 
+  def common_logging_route_name
+    Settings.container_logging.common_logging_route
+  end
```

Comment:
Think about going with `Settings.container_logging.try(:common_logging_route)` and handling a `nil` since I dont know if existing miq instances will re-seed the `config/settings.yml`